src/app/features/characters/components/character-info/character-info.component.ts

```diff
@@ -0,0 +1,25 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-character-info',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './character-info.component.html',
+  styleUrl: './character-info.component.scss',
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class CharacterInfoComponent {
+  @Input() character!: any;
+  name = '';
+  url = '';
+  info = '';
+  comics: any = [];
+
+  ngOnChanges() {
+    this.name = this.character?.name;
```

Comment:
let's discuss, seems you do not need these lots of variables

Reply:
I will come to you with this case, it difficult for me to change it, I tried...